style/javascript/.eslintrc.json

```
@@ -104,7 +104,7 @@
     ],
     "array-bracket-spacing": [
       2,
-      "always"
+      "never"
```

TBH, I want no rule on this, both in JS and in Ruby. We're smarter than a linter at knowing how to make arrays readable.

How do you feel about this?

```suggestion
      "never", { "objectsInArrays": true, "arraysInArrays": true }
```

For those reading along at home, here are the docs: https://eslint.org/docs/rules/array-bracket-spacing

---

I definitely prefer consistent no spaces inside arrays regardless of the content. No spaces is also consistent with Prettier which seems to be what we're recommending too.

---

How would you feel about

```suggestion
      "off"
```

instead? Let the dev figure out what's best.

---

Having worked with Prettier, elm-format, brittany, and rufo over the past few months, I'm in favor of a formatter handle this and make the decision for me. I don't have a preference of whether this is set to `"always"` or `"never"`, although if I had to pick, I'd lean towards `"never"`